docs/src/main/paradox/errorhandling.md

```diff
@@ -28,6 +28,11 @@ Scala
 Java
 : @@ snip [stopping](../../test/java/sample/javadsl/StreamWrapperActor.java) { #errorHandlingSupervisor }
 
+In order to handle **Consumer fails due to WakeupExceptions** while using the ClosedGraph.
```

Comment:
For users not already aware of the problem this might seem a bit cryptic. Perhaps add a little more background here on when/why this error can occur and how you're handling it.